website/templates/public/pages/faq.mako

```diff
@@ -104,10 +104,10 @@
 <p>Send an <a href="mailto:support@osf.io">email</a> with your request. We’ll get that taken care of for you.</p>
 
 <h3>How can I license my data/code/etc.?</h3>
-<p>If you’d like to attach a license to your materials hosted on the OSF, you can put this information in your project’s wiki or upload a README file. Typically, users wish to license their materials using Creative Commons licenses. Information about Creative Commons licenses can be found <a href=”https://creativecommons.org/licenses/”>here.</a> </p>
+<p>If you’d like to attach a license to your materials hosted on the OSF, you can put this information in your project’s wiki or upload a README file. Typically, users wish to license their materials using Creative Commons licenses. Information about Creative Commons licenses can be found <a href="https://creativecommons.org/licenses/">here.</a> </p>
```

Comment:
Looks like there's still a smart singlequote in here. I will fix that when I merge.